datafusion/physical-plan/src/joins/join_filter.rs

```diff
@@ -30,15 +30,15 @@ pub struct JoinFilter {
     /// Column indices required to construct intermediate batch for filtering
     pub(crate) column_indices: Vec<ColumnIndex>,
     /// Physical schema of intermediate batch
-    pub(crate) schema: Schema,
+    pub(crate) schema: Arc<Schema>,
```

Comment:
I think this is more commonly referred to as `SchemaRef` (it is the same thing internally)

```suggestion
    pub(crate) schema: SchemaRef,
```